clusterloader2/testing/load/daemonset.yaml

```diff
@@ -28,5 +28,7 @@ spec:
           requests:
             cpu: 10m
             memory: "10M"
+      tolerations:
+      - operator: Exists
```

Comment:
Not sure if I follow this. FTR, DS has some tolerations added automatically, see https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#taints-and-tolerations

Reply:
it can add add tolerations automatically with only the specified ones. If I have a master tained other than these, then it will fail.